WordPress/Tests/WP/DeprecatedParametersUnitTest.inc

```diff
@@ -0,0 +1,53 @@
+<?php
+
+// all will give an ERROR.
+
+get_last_updated( 'deprecated' );
+xfn_check( '', '', 'deprecated' );
+convert_chars( '', 'deprecated' );
+comments_link( 'deprecated', 'deprecated' );
+comments_number( '', '', '', 'deprecated' );
+the_author( 'deprecated', 'deprecated' );
+wp_upload_bits( '', 'deprecated' );
+get_the_author( 'deprecated' );
+the_author_posts_link( 'deprecated' );
+add_option( '', '', 'deprecated' );
+get_user_option( '', '', 'deprecated' );
+the_attachment_link( '', '', 'deprecated' );
+trackback_rdf( 'deprecated' );
+trackback_url( 'deprecated' );
+wp_install( '', '', '', '', 'deprecated' );
+discover_pingback_server_uri( '', 'deprecated' );
+load_plugin_textdomain( '', 'deprecated' );
+wp_get_http_headers( '', 'deprecated' );
+safecss_filter_attr( '', 'deprecated' );
+wp_get_sidebars_widgets( 'deprecated' );
+get_delete_post_link( '', 'deprecated' );
+is_email( '', 'deprecated' );
+update_user_status( '', '', '', 'deprecated' );
+update_blog_option( '', '', '', 'deprecated' );
+wp_notify_postauthor( '', 'deprecated' );
+wp_new_user_notification( '', 'deprecated' );
+wp_new_user_notification( '', $variable );
+wp_new_user_notification( '', function_name() );
+
+
+// all will give an WARNING.
+
+get_wp_title_rss( 'deprecated' );
+wp_title_rss( 'deprecated' );
+unregister_setting( '', '', '', 'deprecated' );
+
+// all will be OK.
```

Comment:
Maybe add some info on why these are ok ? Is it because they have the default value ? or because the params aren't deprecated ? or ... ?